docs/development/quickstart.md

```diff
@@ -0,0 +1,310 @@
+
+       You can use `choco` for that
+
+       ```cmd
+       choco install -y --limit-output winfsp --version=1.11.22176
+       ```
+
+       Or you can use the installer <https://winfsp.dev/rel/>
+
+       > We haven't tried this method so if you can provide feedback if you use it.
+
+     - On MacOS: you need to install `macfuse`
+
+       ```shell
+       brew install --cask macfuse
+       ```
+
+     - On Linux: you need to have installed `libfuse2`
+
+3. Initialize a poetry env
+
+   ```shell
+   poetry install --extra 'core backend'
+   ```
+
+4. Start a shell with the initialized virtual env
+
+    ```shell
+    poetry shell
+    ```
+
+5. To run parsec do
+
+    ```shell
+    poetry run parsec
+    ```
+
+Happy Hacking 🐍
+
+### Run the python tests
+
+Run the tests with `pytest`:
+
+```shell
+poetry run py.test tests
+```
+
+On top of that, multiple options are available:
+
+| Option              | Description                                               |
+| ------------------- | --------------------------------------------------------- |
+| ``--runmountpoint`` | Include mountpoint tests                                  |
+| ``--rungui``        | Include GUI tests                                         |
+| ``--runslow``       | Include slow tests                                        |
+| ``--postgresql``    | Use PostgreSQL in the backend instead of a mock in memory |
+| ``-n 4``            | Run tests in parallel (here `4` jobs)                     |
+
+Note you can mix&match the flags, e.g.
+
+```shell
+py.test tests --runmountpoint --postgresql --runslow -n auto
+```
+
+If you want to run GUI test, it is a good idea to install ``pytest-xvfb`` in order to
+hide the Qt windows when running the GUI tests
+
+```shell
+apt install xvfb
+pip install pytest-xvfb
+```
+
+### Run Parsec in local
+
+You can use the ``run_testenv`` scripts to easily create a development environment:
+
+On linux and MacOS:
+
+```shell
+. ./tests/scripts/run_testenv.sh
+```
+
+On Windows:
+
+```cmd
+.\tests\scripts\run_testenv.bat
+```
+
+This script will:
+
+- Start a development backend server with in-memory storage
+- Configure environment variables to isolate the development environment from
+  your global Parsec configuration
+- Create a default organization
+- Create multiple users and devices for this organization
+
+## Hacking the Web client (front)
+
+In addition to the [shared requirements](#shared-requirements), you need to install [`Node 18.12.0`](https://nodejs.org/en/download/releases).
+
+> You could use [`nvm`](https://github.com/nvm-sh/nvm) to manage multiple node version
+> or [`nvm-windows`](https://github.com/coreybutler/nvm-windows) for `Windows`.
+
+1. Install [`Qt5`](https://www.qt.io/).
+
+   The `python` application use `PyQt5`, you need to have `qt5` installed because even if you don't need that part.
+   The build system will still require it (and).
```

Comment:
```suggestion
   The `python` application uses `PyQt5`, you need to have `qt5` installed because
   the build system will require it even if you don't modify Python code.
```

Reply:
Why specifying `even if you don't modify Python code` ?

Reply:
Because that's what I understood when I first read it ^^
Did I changed the original meaning?

Reply:
For me the part with `even ...` is superfluous